splunk/src/main/java/com/splunk/Index.java

```diff
@@ -358,10 +360,10 @@ public String getMaxDataSize() {
     /**
      * Returns the maximum number of hot buckets that can exist for this index.
      *
-     * @return The maximum number of hot buckets.
+     * @return The maximum number of hot buckets or "auto" (which means 3).
      */
-    public int getMaxHotBuckets() {
-        return getInteger("maxHotBuckets");
+    public String getMaxHotBuckets() {
+        return getString("maxHotBuckets");
```

Comment:
- [ ] Let's capture this in the RELEASE_NOTES.md since it is developer facing

Reply:
This point and all following points has to be considered when we draft new **github release**. Is it correct ?
So, do we need RELEASE_NOTES.md file for the same ?